Cargo.toml

```diff
@@ -44,3 +44,6 @@ json = "0.11"
 [features]
 default = []
 clippy = ["clippy_lints"]
+
+[patch.crates-io]
```

Comment:
This is just to get the ci builds to pass until `languageserver-types` merges their PR and releases a version.